lib/verbalize/action.rb

```diff
@@ -24,8 +24,16 @@ def optional_inputs
         @optional_inputs || []
       end
 
+      def default_inputs
+        (@defaults || {}).keys
```

Comment:
So unless I'm missed some place, looks like this is only used internally, and could be private. I don't consider it a blocker though.